src/SimulatorView/SimulatorView.tsx

```diff
@@ -0,0 +1,15 @@
+import React, { FunctionComponent } from "react";
```

Comment:
From the gitter chat:
Only possible suggestion would be to make the "SimulatorView" folder more generic - maybe something like "Layout" or "Views"? Totally up to you.